chart/env/prod.yaml

```diff
@@ -93,6 +93,9 @@ parquetPersistence:
 duckdbPersistence:
   existingClaim: "datasets-server-duckdb-pvc"
 
+statisticsPersistence:
+  existingClaim: "datasets-server-statistics-pvc"
```

Comment:
Is there something needed on infra to create this pvc?

Reply:
cc @AndreaFrancis

Reply:
Yes, I think we need these assets to be created for the new claim like https://github.com/huggingface/infra/blob/main/projects/hub/kubernetes_app_datasets_server_duckdb_efs.tf. Maybe @rtrompier can help us with it?

Reply:
New volume is created and deployed on production (`hub-prod-us-east-1` cluster).